src/components/common/OverviewItemCard.tsx

```diff
@@ -43,7 +43,7 @@ export default function OverviewItemCard({
         display: 'flex',
         height: '100%',
         justifyContent: 'space-between',
-        padding: theme.spacing(3),
+        padding: 0,
```

Comment:
This will break other places in the app where this component is used (e.g. Accession view). I added the `className` prop to allow overriding this.